Saved views and more (#896)
avohq/docs: pages/workspace/add-events-from-inspector.mdx

import Link from '../../components/Link';

> The Inspector compares the events and properties coming in to the tracking plan and detects if there are any of the incoming missing from the tracking plan. Event and properties can both be added to the tracking plan from the <Link passHref href="/workspace/inspector/inspector-issues-view"><a>Issues view</a></Link> and the <Link passHref href="/workspace/inspector/inspector-events-view"><a>Events view</a></Link>.

Reviewer: ```suggestion
The Inspector compares the injested events and properties against the tracking plan and detects if there are any properties missing from the tracking plan. Unknown events and properties can be added to the tracking plan from both the <Link passHref href="/workspace/inspector/inspector-issues-view"><a>Issues view</a></Link> and the <Link passHref href="/workspace/inspector/inspector-events-view"><a>Events view</a></Link>.
```

## Event not in Tracking Plan

### Issues view

> When an event is missing from the tracking plan, the issue details of the issue view will offer you to add it to the tracking plan right from there. Click the "Add to Tracking Plan" button and the event and all it's properties are created.

Reviewer: ```suggestion
When an event is missing from the tracking plan the issue details view will offer you to quickly add it to the tracking plan. Click the "Add to Tracking Plan" button and the event and all it's properties are created.
```
Although this is not true today, is it? We're not displaying issuse for unknown events?

Author: We're not displayin the issue for missing events, but you can still add the event from the issue details if it is not in the tracking plan.

<center>
  <img
    src={require('../../images/inspector/add-event-from-issue-details.png')}
    width="100%"
    alt='Image shows issue details with "Add to tracking plan" highlighted for the event.'
  />
</center>

### Events view

> If the Inspector detects an Event that is not in your tracking plan, it will be highlighted in the events view with a red tracking plan icon next to the name, along with an "Add" button to add it to your tracking plan. The slide-over in the events view also has an option to add the event right from there. Click either button to add the event and all it's properties to the tracking plan.

Reviewer: ```suggestion
If the Inspector detects an event that is not in your tracking plan, it will be highlighted with a red tracking plan icon next to the name, along with an "Add" button to add it to your tracking plan. The events view sidebar also has an option to add the event right from there. Click either button to add the event and all it's properties to the tracking plan.
```

    width="100%"
    alt='Image shows the Inspector events view with "Add" and "Add event to tracking plan" buttons highlighted.'
  />
</center>

## Property not in tracking plan 

### Issues view

If the Inspector detects a property that is not in you tracking plan, the issue type "Unexpected Property" is presented. The issues view can be filtered by issue type, so when that filter is applied you get a good overview over all unexpected properties. 

<center>
  <img
    src={require('../../images/inspector/filter-by-unexpected-property.png')}
    width="100%"
    alt='Image shows the Inspector issues view filters with Unexpected property filter selected.'
  />
</center>

In the issue details, you will always be able to see whether the event or property is currently in the tracking plan and you get an option to add it if it's not. Click the "Add to tracking plan" button by the property name to add it to your tracking plan.

<center>
  <img
    src={require('../../images/inspector/add-property-from-issue-details.png')}
    width="100%"
    alt='Image shows issue details with "Add to tracking plan" highlighted for the property.'
  />
</center>

### Events view

> Selecting an event in the events view will expand down such that all the properties that were detected in this particular event are shown. If a property is missing from the tracking plan for that particular event, it is highlighted with a red tracking plan icon next to the property name, along with an "Add" button to add it to your tracking plan. The slide over in the events view also has an option to add the property to the tracking right from there. Click either button to add the event and all it's properties to the tracking plan. There are two cases for adding property to the event in the tracking plan:

Reviewer: ```suggestion
Clicking no the event name cell will expand it such that all the properties that were detected in this particular event are shown. If a property is missing from the tracking plan for that particular event, it is highlighted with a red tracking plan icon next to the property name, along with an "Add" button to add it to your tracking plan. The events view sidebar also has an option to add the property to the tracking right from there. Click either button to add the event and all it's properties to the tracking plan. There are two cases for adding property to the event in the tracking plan:
```
